HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -91,6 +90,27 @@ string custodianCode
             };
             SendEmail(emailModel);
         }
```

Comment:
You should have had a new line after this brace and before the next method, but the private method is going to be moved to the bottom anyway.

I find that some of the inline code info that IDEs put in these days can obscure how many new lines there are about the place. I always turn off those sorts of hints, but if you want to leave them on then it's worth being extra vigilant on new lines.

Reply:
Yeah, I need to definitely be more careful with this. One of the problems is I find the little "usages" bit quite useful so I don't want to disable it, but it is quite annoying that it gets in the way too. 
Is there a way to have it only show up some of the time? ex. while the caret is inside that block

Reply:
I'm not aware of a way to have it only show up some of the time. I always just turn them off!